my_pyscf/grad/lasscf.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python
+# Copyright 2014-2019 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Qiming Sun <osirpt.sun@gmail.com>
+#
+
+'''
+CASSCF analytical nuclear gradients
+
+Ref.
+J. Comput. Chem., 5, 589
+
+MRH: copied from pyscf.grad.casscf.py on 12/07/2019
+Contains my modifications for SA-CASSCF gradients
+1. Generalized Fock has nonzero i->a and u->a
+2. Memory footprint for differentiated eris bugfix
+'''
+
+'''
+SV: added and modified for casscf.py for lasscf nuclear gradients
+Also check lasscf_sync_o0.py where a nuc_grad_method has been defined with a Gradients class!
```

Comment:
So why isn't `lasscf_sync_o0.py` in this commit?

Reply:
The file is added now.